### PR TITLE
all cases run locally

### PR DESCRIPTION
**Pull request by sisiz**
all of the cases are running locally.

### CODE REVIEW

File: tutor/test_cc1_03_ContentPreparationAndImport.py [hunk 1]
```diff
@@ -124,8 +174,61 @@ def test_admin_import_content_into_tutor_7604(self):
         self.ps.test_updates['passed'] = False
 
         # Test steps and verification assertions
-        raise NotImplementedError(inspect.currentframe().f_code.co_name)
-
+        self.content.login(
```
Comment (gregfitch) [inline tutor/test_cc1_03_ContentPreparationAndImport.py]: This should use an Admin() object that uses the ContentQA().driver for clarity.

File: tutor/test_cc1_03_ContentPreparationAndImport.py [hunk 2]
```diff
@@ -37,14 +38,14 @@ class TestContentPreparationAndImport(unittest.TestCase):
...
-        raise NotImplementedError(inspect.currentframe().f_code.co_name)
-
         self.ps = PastaSauce()
         self.desired_capabilities['name'] = self.id()
         self.content = ContentQA(
             use_env_vars=True,
             pasta_user=self.ps,
-            capabilities=self.desired_capabilities
+            capabilities=self.desired_capabilities,
+            username=os.getenv('CONTENT_USER'),
```
Comment (gregfitch) [inline tutor/test_cc1_03_ContentPreparationAndImport.py]: Let me investigate this...it shouldn't require anything else besides use_env_vars, pasta_user and capabilities.

File: tutor/test_cc1_03_ContentPreparationAndImport.py [hunk 3]
```diff
@@ -124,8 +174,61 @@ def test_admin_import_content_into_tutor_7604(self):
         self.ps.test_updates['passed'] = False
 
         # Test steps and verification assertions
-        raise NotImplementedError(inspect.currentframe().f_code.co_name)
-
+        self.content.login(
+            username=os.getenv('ADMIN_USER'),
+            password=os.getenv('ADMIN_PASSWORD'),
+        )
```
Comment (gregfitch) [inline tutor/test_cc1_03_ContentPreparationAndImport.py]: This should use an Admin() object that uses the ContentQA().driver for clarity.

File: tutor/test_cc1_03_ContentPreparationAndImport.py (hunk 2 above)
Comment (sisiz) [inline tutor/test_cc1_03_ContentPreparationAndImport.py]: I was being stupid and commented out use_env_vars=True. Just tested it there's no issue with this

File: tutor/test_cc1_03_ContentPreparationAndImport.py (hunk 3 above)
Comment (sisiz) [inline tutor/test_cc1_03_ContentPreparationAndImport.py]: fixed this

**Pull request merged**